src/main/java/uk/gov/companieshouse/search/api/mapper/ApiToResponseMapper.java

```diff
@@ -18,6 +19,8 @@ public ResponseEntity map(ResponseObject responseObject) {
                 return ResponseEntity.status(FOUND).body(responseObject.getData());
```

Comment:
where is the apiToReponseMapper elements for UPDATE_REQUEST_ERROR and UPSERT_ERROR?

Reply:
Good spot must have missed them.